server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/cluster/ProcessInfoProvider.java

```diff
@@ -25,38 +25,37 @@
 import org.sonar.api.Startable;
 import org.sonar.api.ce.ComputeEngineSide;
 import org.sonar.api.server.ServerSide;
-import org.sonar.process.cluster.info.ProcessInfoProvider;
-import org.sonar.process.cluster.info.ProcessInfoProviderRef;
 import org.sonar.process.systeminfo.Global;
 import org.sonar.process.systeminfo.SystemInfoSection;
 import org.sonar.process.systeminfo.protobuf.ProtobufSystemInfo;
 
 @ServerSide
 @ComputeEngineSide
-public class ProcessInfoProviderImpl implements ProcessInfoProvider, Startable {
+public class ProcessInfoProvider implements Startable {
 
+  /** Used for Hazelcast's distributed queries in cluster mode */
+  private static ProcessInfoProvider INSTANCE;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "INSTANCE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3008)